spec/services/failsafe_draft_preview_service_spec.rb

```diff
@@ -21,7 +21,13 @@
     end
 
     context "when there are pre-preview issues" do
-      let(:edition) { create(:edition, summary: "new\nline", revision_synced: true) }
+      let(:document_type) do
+        build :document_type, contents: [DocumentType::TitleAndBasePathField.new]
+      end
+
+      let(:edition) do
+        create(:edition, document_type: document_type, title: "", revision_synced: true)
```

Comment:
Looks like `document_type` can be a variable scoped to this block, rather than a `let` in its own right.

Reply:
Oh yeah, good point - updated.